### ISSUE TITLE
I must be doing something wrong..

### ISSUE THREAD

**Issue by steffenschumacher** [asottile/pyupgrade#337]
I'm struggling to make pyupgrade fix % strings to format, but with no luck:
```
(venv36) $ echo 'x = "found cached auth object for user %s" % username' | pyupgrade -
x = "found cached auth object for user %s" % username
(venv36)$ pip freeze | grep pyupgrade
pyupgrade==2.7.2
```

Where I was expecting:
```
x = "found cached auth object for user {}".format(username)
```

I must be doing something obvious wrong - and it doesn't appear to be related to type of quote:
```
(venv36) $ echo "x = 'found cached auth object for user %s' % username" | pyupgrade -
x = 'found cached auth object for user %s' % username
```
Any cluepons me?

**Comment by asottile** [comment]
see #256 and #78

tl;dr this is part of the reason that % formatting is bad, it's impossible to know statically whether that's intended to be rewritten as `'{}'.format(x)` or `{}.format(*x)` because `'%s' % x` could be either depending on the type of x

**Comment by asottile** [comment]
(this is only for 1-ary % format in case that isn't clear from the linked issues)
